libraries/Arduino_CAN/src/R7FA4M1_CAN.cpp

```diff
@@ -187,6 +187,18 @@ void R7FA4M1_CAN::end()
   R_CAN_Close(&_can_ctrl);
 }
 
+
+void R7FA4M1_CAN::setMailboxMask(int mailbox_group, uint32_t mask)
+{
+	_can_mailbox_mask[mailbox_group] = mask;
+}
+
+void R7FA4M1_CAN::setMailboxID(int mailbox, int id)
+{
+	_can_mailbox[mailbox].mailbox_id = id;
```

Comment:
Use the same level of indentation for the rest of the code, that is "2 spaces" per indentation level.